airbyte-integrations/connectors/source-amazon-ads/unit_tests/test_report_streams.py

```diff
@@ -556,3 +559,42 @@ def test_read_incremental_with_records_start_date(config):
         records = list(read_incremental(stream, state))
         assert state == {"1": {"reportDate": "20210104"}}
         assert {r["reportDate"] for r in records} == {"20210103", "20210104", "20210105", "20210106"}
+
+
+@pytest.mark.parametrize("state_filter", [["enabled", "archived", "paused"], ["enabled"], None])
+def test_display_report_stream_campaign_state_filer(mocker, config, state_filter):
+    profiles = make_profiles()
+    mocker.patch.object(SponsoredDisplayCampaigns, "state_filter", new_callable=mocker.PropertyMock, return_value=state_filter)
+
+    stream = SponsoredDisplayCampaigns(config, profiles)
+    params = stream.request_params(stream_state=None, stream_slice=None, next_page_token=None)
+    if "stateFilter" in params:
+        assert params["stateFilter"] == ",".join(state_filter)
+    else:
+        assert state_filter is None
+
+
+@pytest.mark.parametrize("state_filter", [["enabled", "archived", "paused"], ["enabled"], None])
+def test_display_report_stream_product_campaign_state_filer(mocker, config, state_filter):
```

Comment:
This test idential to previous test_display_report_stream_campaign_state_filer
can we merge all this 3 tests to one and  parameterize class name: SponsoredDisplayCampaigns, SponsoredProductCampaigns, SponsoredBrandsCampaigns